package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "koa-middlewares",
-  "version": "2.1.0",
+  "version": "2.2.0",
```

Comment:
Nice. I'm waiting for your your package's publication on npmjs.com. Thanks for your work.